docs/se/introduction/01_introduction.adoc

```diff
@@ -38,37 +38,58 @@ of Netty and uses a straight forward request routing API and supports a
 number of additional Helidon features:
```

Comment:
It makes sense to add links to the corresponding docs for each feature in the table.